webapi/src/main/scala/org/knora/webapi/http/status/ApiStatusCodesV2.scala

```diff
@@ -22,21 +22,25 @@ object ApiStatusCodesV2 {
    */
   def fromException(ex: Throwable): StatusCode =
     ex match {
-      // Subclasses of RequestRejectedException (which must be last in this group)
+      // Subclasses of RequestRejectedException
       case NotFoundException(_)           => StatusCodes.NotFound
       case ForbiddenException(_)          => StatusCodes.Forbidden
       case BadCredentialsException(_)     => StatusCodes.Unauthorized
       case DuplicateValueException(_)     => StatusCodes.BadRequest
       case OntologyConstraintException(_) => StatusCodes.BadRequest
       case EditConflictException(_)       => StatusCodes.Conflict
-      case RequestRejectedException(_)    => StatusCodes.BadRequest
       case BadRequestException(_)         => StatusCodes.BadRequest
+      case ValidationException(_, _)      => StatusCodes.BadRequest
+      case RequestRejectedException(_)    => StatusCodes.BadRequest
+      // RequestRejectedException must be last in this group
```

Comment:
Why it must be the last one?

```suggestion
      // RequestRejectedException must be the last one in this group
```

Reply:
That's what it said before ;) I guess because it is the most generic one of all Exceptions in this list. If it came before another one, the other one (more specific one) would never be matched.